### PR TITLE
Integrating x471 changes into LAMP master repo

### PR DESCRIPTION
**Pull request by Mbosum**
I made some changes to the LAMP motion configuration to integrate it into the most recent experiment setup (x454). This is the most up-to-date running LAMP motion code. The most important change was the integration of the additional Flow Cell Manipulator with 4 axes which were added to the NC:

-Flow Cell X
-Flow Cell Y
-Flow Cell Z
-Flow Cell Theta

However, since no new axes were added, the new NC axes had to be linked to the terminals previously allocated to Gas Nozzle XYZ and M10 in the IO. This is temporary for the moment, but if it looks like it will be a more stable setup for the future, the names in the IO can be changed too.

I have also created a release for the configuration (x471) immediately before the Flow Cell Manipulator was installed.

### CODE REVIEW

File: lcls-plc-lamp-motion/_Config/NC/Axes/Axis 11.xti [hunk 1]
```diff
@@ -0,0 +1,1545 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
```
Comment (ZLLentz) [inline lcls-plc-lamp-motion/_Config/NC/Axes/Axis 11.xti:1]: I think these Axis 11, 12, and 13 files are added but unused

File: lcls-plc-lamp-motion/lamp_motion/POUs/Main.TcPOU [hunk 2]
```diff
@@ -119,13 +146,17 @@ END_VAR]]></Declaration>
 	mcSetPosition[7](Axis:=M7.Axis, Position:=fGasPaddleXPos, Mode:=FALSE, Execute:=TRUE);
 	mcSetPosition[8](Axis:=M8.Axis, Position:=fGasPaddleyPos, Mode:=FALSE, Execute:=TRUE);
 	mcSetPosition[9](Axis:=M9.Axis, Position:=fGasPaddleZPos, Mode:=FALSE, Execute:=TRUE);
-	mcSetPosition[10](Axis:=M10.Axis, Position:=fM10Pos, Mode:=FALSE, Execute:=TRUE);
+	mcSetPosition[10](Axis:=M10.Axis, Position:=fFLowCellXPos, Mode:=FALSE, Execute:=TRUE);
+	mcSetPosition[11](Axis:=M11.Axis, Position:=fFLowCellYPos, Mode:=FALSE, Execute:=TRUE);
+	mcSetPosition[12](Axis:=M12.Axis, Position:=fFLowCellZPos, Mode:=FALSE, Execute:=TRUE);
+	mcSetPosition[13](Axis:=M13.Axis, Position:=fFLowCellThetaPos, Mode:=FALSE, Execute:=TRUE);
```
Comment (ZLLentz) [inline lcls-plc-lamp-motion/lamp_motion/POUs/Main.TcPOU:152]: it's worth noting that the motion library timing bugs related to MC_SetPosition have been resolved. If these don't set the positions properly due to the the same bug it'd be worth to switch to using the library support.

**Review by ZLLentz — APPROVED** [review]
Left some questions/comments/suggestions but none of them are blocking

File: lcls-plc-lamp-motion/lamp_motion/POUs/Main.TcPOU (hunk 2 above)
Comment (klauer) [reply lcls-plc-lamp-motion/lamp_motion/POUs/Main.TcPOU:152]: Using the library's built-in support would significantly simplify this logic, I believe.

**Review by klauer — APPROVED** [review]
I'd recommend you rebuild for ads-ioc 0.5.0 for this repo too (this is on 0.4.1).

Everything else seemed fine to me 👍

**Comment by Mbosum** [comment]
For upgrading the ads-ioc version to 0.5.0, and using the motion library's built-in support for MC_SetPosition, would it make sense to do these in a separate PR? That way I can test them on my own forked branch first and have this previously running code as a backup.

File: lcls-plc-lamp-motion/lamp_motion/POUs/Main.TcPOU (hunk 2 above)
Comment (ZLLentz) [reply lcls-plc-lamp-motion/lamp_motion/POUs/Main.TcPOU:152]: Yes, exactly.

**Comment by ZLLentz** [comment]
Ken and I both think this PR is :+1: to go, it is very sensible to merge and consider the suggestions for a separate PR. It's up to your discretion (we both clicked the approve button)

**Pull request merged**